BangazonAPI/Controllers/PaymentTypeController.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using BangazonAPI.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PaymentTypeController : ControllerBase
+    {
+        // GET: api/<PaymentTypeController>
+
+        //Following code allows access to SQL database
+
+        private readonly IConfiguration _config;
+
+        public PaymentTypeController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public SqlConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT Id, AcctNumber AS AccountNumber, Name AS PaymentType, CustomerId FROM PaymentType";
+
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    List<PaymentType> paymentTypes = new List<PaymentType>();
+
+                    while (reader.Read())
+                    {
+
+                        if (!paymentTypes.Any(p => p.Id == reader.GetInt32(reader.GetOrdinal("Id"))))
```

Comment:
This line of code could be deleted-- this is basically checking to see if the payment type is already in the list before we add it, but because we're just doing a straight up select query with no joins we should never have a situation where the same payment type could be added more than once